bgpd/bgp_vty.c

```diff
@@ -1413,22 +1419,16 @@ DEFUN (no_auto_summary,
 }
 
 /* "router bgp" commands. */
-DEFUN_NOSH (router_bgp,
-       router_bgp_cmd,
-       "router bgp [(1-4294967295)$instasn [<view|vrf> VIEWVRFNAME]]",
-       ROUTER_STR
-       BGP_STR
-       AS_STR
-       BGP_INSTANCE_HELP_STR)
+DEFPY_NOSH(
+	router_bgp, router_bgp_cmd,
+	"router bgp [<A.B|(1-4294967295)>$instasn [<view|vrf> VIEWVRFNAME$name]]",
+	ROUTER_STR BGP_STR ASDOT_STR AS_STR BGP_INSTANCE_HELP_STR)
```

Comment:
Please keep an old formatting for DEFUN.

Reply:
ok